packages/gafl-webapp-service/src/pages/contact/contact/__tests__/update-transaction.spec.js

```diff
@@ -258,5 +300,34 @@ describe('contact > update-transaction', () => {
         })
       )
     })
+
+    it('should set email, preferredMethodOfReminder and preferredMethodOfConfirmation in cache when how-contacted is none and preferredMethodOfNewsletter is email', async () => {
```

Comment:
This is looking similar to previous tests - is there any way we could combine them into a single test using `each` to vary the arguments?

Reply:
That was a fun exercise in abstraction :) Done!